pkg/cmd/kafka/acl/admin/admin.go

```diff
@@ -83,7 +83,7 @@ func NewAdminACLCommand(f *factory.Factory) *cobra.Command {
 			}
 
 			// user and service account should not allow wildcard
-			if userID == aclutil.Wildcard || serviceAccount == aclutil.Wildcard || userID == aclutil.AllAlias || serviceAccount == aclutil.AllAlias {
+			if userID == sdk.Wildcard || serviceAccount == sdk.Wildcard || userID == sdk.AllAlias || serviceAccount == sdk.AllAlias {
```

Comment:
If you look at this line, it is very difficult to understand what `sdk` relates to. `aclutil` is much more self descriptive.

`<thing>util` is a common pattern in Go: https://pkg.go.dev/io/ioutil

Reply:
I have followed `<thing>util` for utility methods at top level, i.e `pkg/kafkautil`, `pkg/serviceregistryutil`. I was hoping to follow a different naming convention for utils specific to commands, maybe `aclcmdutil`, `topiccmdutil` could be better alternative? Wdyt?

Reply:
I think it is all about import path.

if util/sdk/common has subpackages then we have nice pattern - single name used across all cli packages with specific sub packages

Reply:
Quick look at other projects confirms what enda said. Havents seen packages with common subpackages being used in go

Reply:
Although our case is different as people might want import sdk etc. so it is self documenting change. etc

Reply:
Yeah, the import path does not matter a bit, what is important is the name of the package which will be used in the code.

Reply:
> I have followed `<thing>util` for utility methods at top level, i.e `pkg/kafkautil`, `pkg/serviceregistryutil`. I was hoping to follow a different naming convention for utils specific to commands, maybe `aclcmdutil`, `topiccmdutil` could be better alternative? Wdyt?

Yes I like that!.